Clarification About GitOps Command Execution on Pushed Commits (#1911)
tektoncd/pipelines-as-code: docs/content/docs/guide/gitops_commands.md

For restarting a specific pipeline run:
2. Use `/retest <pipelinerun-name>` or `/test <pipelinerun-name>` within your commit message. Replace `<pipelinerun-name>` with the specific name of the pipeline run you want to restart.

The GitOps command triggers a PipelineRun only on the latest commit (HEAD) of the branch and does not work on older commits.

**Note:**

When executing `GitOps` commands on a commit that exists in multiple branches within a push request, the branch with the latest commit will be used.

This means:

> 1. If a user specifies commands like `/retest` or `/test` without specifying branch name in a comment on branch, the test will automatically be performed on the **default branch** (e.g. main, master etc.) of the repository.

Reviewer: ```suggestion
1. When a user comments with commands like `/retest` or `/test` on a branch without specifying a branch name, the test will automatically run on the **default branch** (e.g., main, master) of the repository.```

For restarting a specific pipeline run:
2. Use `/retest <pipelinerun-name>` or `/test <pipelinerun-name>` within your commit message. Replace `<pipelinerun-name>` with the specific name of the pipeline run you want to restart.

> The GitOps command triggers a PipelineRun only on the latest commit (HEAD) of the branch and does not work on older commits.

Reviewer 2: Can you add this also under NOTE

Author: is it needed if it is above note? it would be two time.
